webapp/components/survey/SurveyForm/nodeDefs/components/types/nodeDefBoolean.js

```diff
@@ -26,7 +26,7 @@ const NodeDefBoolean = (props) => {
         className="btn btn-s btn-transparent flex-center"
         data-value={value}
         aria-disabled={disabled || !canEditRecord || readOnly}
-        onClick={() => updateNode(nodeDef, node, checked ? null : value)}
+        onClick={() => (checked ? null : updateNode(nodeDef, node, value))}
```

Comment:
does this prevent resetting a boolean node?

Reply:
I changed the test to avoid the reset